css/order.css

```diff
@@ -69,53 +69,59 @@
   flex: 2;
 }
 
-.bnomics-altcoin-pane {
-  align-items: center;
-  flex: 1;
-  text-align: center;
+.bnomics-grey {
+  color:grey;
 }
 
 .bnomics-address {
+  margin-top: 10px;
   position:relative;
   margin:auto !important;
   width: 90% !important;
```

Comment:
The 90% width restraint can be removed as it is causing the address and amount inputs to vary in lengths.
![EDD Spacing](https://user-images.githubusercontent.com/39666372/105028368-9d257900-5a59-11eb-93d3-4a7bb61cf4f8.png)